hr_employee_id/models/res_config.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Business Applications
+#    Copyright (C) 2004-2012 OpenERP S.A. (<http://openerp.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, SUPERUSER_ID
+
+
+class human_resources_configuration(models.TransientModel):
+    _inherit = 'hr.config.settings'
+
+    employee_id_gen_method = fields.Selection((('random', 'Random'),
+                                               ('sequence', 'Sequence')), string="ID Generation Method", default='random',
```

Comment:
@tarzan0820 : use this: fields.Selection(
        [
            ('random', 'Random'),
            ('sequence', 'Sequence'),
        ],
.....